src/components/OrderBreakdown/LineItemBasePriceMaybe.js

```diff
@@ -22,15 +23,14 @@ const LineItemBasePriceMaybe = props => {
   // It should be one of the following: 'line-item/night, 'line-item/day', 'line-item/hour', or 'line-item/item'
   // These are defined in '../../util/types';
   const unitPurchase = lineItems.find(item => item.code === code && !item.reversal);
-
-  const quantity = unitPurchase ? unitPurchase.quantity.toString() : null;
   const unitPrice = unitPurchase ? formatMoney(intl, unitPurchase.unitPrice) : null;
   const total = unitPurchase ? formatMoney(intl, unitPurchase.lineTotal) : null;
+  const displayQuantity = quantity ? quantity : estimatedLineItem;
```

Comment:
The latter would use `estimatedLineItem` if `quantity` has falsey value. The former function almost the same way. However, the former can be used when we want to use other value instead of `quantity` for the true case. And for some reason if we want to use `quantity` only if it is falsey (like `quantity? estmatedLineItem : quantity`). But in this case, the latter (i.e. the OR operator) is more suitable.